docs/adr/0001-retiring-licence-finder.md

```diff
@@ -0,0 +1,89 @@
+# Retiring Licence Finder
+
+Date: 2022-10-25
+
+## Context
+
+There is a legal requirement for 900 licences need to be created and updated on GOV.UK in the next few months.
+Currently licences are a mainstream document type and the responsibility for maintaining them rests with GDS content designers in Mainstream Publisher.
+
+Developers are also needed to add licences to the search results Licence Finder. Licences are added to this tool by updating a CSV file with the [licence information], e.g. title, description, link, licence identifier and running a rake task. Licences are tagged to [sectors] and [activities] by adding the mapping in a CSV file and running a rake task.
+
+When rendered, licence pages resemble transaction pages, and have an "Apply now" button, if the licence can be applied for online, that takes the user to one of two places: a licence managed by a competent authority on an external website (e.g. TV Licence) or a postcode lookup that allows users to apply for the licence via their local authority. These local authority licences are managed in [Licensify].
+
+Currently users can either find licences in the tool by narrowing down a list of "sectors" and "activities" or by performing a keyword search. Regardless of the approach, the list of results shown to the user is found by querying Elasticsearch with the unique licence identifier assigned to each licence.
+
+If the licence is found in Elasticsearch a link to the licence is displayed to the user.
+
+If the licence cannot be found in Elasticsearch only the title of the licence is displayed to the user. This title may not match the title in the content item.
+
+### <a name="problems"></a> Problems with the current approach
+
+Only a few licences can be considered "mainstream". Most are for a specialist activity. However as licences are managed in Mainstream Publisher only GDS content designers can create licence pages.  As Content Designers across government cannot create licence pages, they have taken to using other Whitehall formats to create licence content. This means that the licences created in non-mainstream formats do not have a licence identifier, and therefore do not appear in the Licence Finder results.
+
+Another reason that not all licences appear in the Licence Finder results is because a developer is needed to updated the CSV files and run a rake task. A lack of clear processes means that this step is easy to miss. This has led to a situation where only approximately 60% of licences in Mainstream Publisher can be found using the Licence Finder.
+
+It is possible for no results to be displayed to the user if they do a text search. For example, "restaurant" doesn't yield any results, even though it is in the name of a sector. This is because the text box search only searches against the list of activities, not sectors or licence names. "Restaurant" only appears in sector names, not activities so the search doesn't display any results.
```

Comment:
Thanks for rewording, however I don't think this paragraph is quite right. I've just done some testing on the actual issue and created a comment accordingly (sectors are queried and substrings are not valid):

It is possible for no results to be displayed to the user if they do a text search. A text box search only queries against the list of activities and sectors, not licence names. The search also doesn't match against a substring of sector or activity names, for example "restaurants" generates results, whereas "restaurant" does not.

Reply:
Ah! Thanks!